package.json

```diff
@@ -152,6 +152,14 @@
     "rimraf": "^3.0.2",
     "semistandard": "^16.0.1"
   },
+  "peerDependencies": { 
+    "node-gyp": "*"
```

Comment:
Could the use of _any_ version cause problems? Perhaps we should limit to the major version number of the minimum `node-gyp` version provided by the bundled version of `npm` for the earliest Node.js we support (npm 6)? If so, this would be `node-gyp` version 5.

Reply:
Ah - the optional peer dependencies need npm v7 :(

Reply:
Which means npm 6 will just complain if node-gyp isn't installed.

Reply:
Thanks, I'd prefer to avoid dealing with support issues about perceived missing dependencies, so it sounds like we should we wait until Node.js 14, and therefore npm v6, reaches EOL.